### PR TITLE
Add `:edit` and alternatives

### PR DESCRIPTION
**Pull request by Fymyte**

### CODE REVIEW

File: keywords.js [hunk 1]
```diff
@@ -434,6 +434,36 @@ const KEYWORDS = {
...
+  EX: {
+    mandat: "ex",
+    opt: "",
+    ignore_comments_after: false,
+  },
+  VISUAL: {
+    mandat: "vi",
+    opt: "sual",
+    ignore_comments_after: false,
+  },
```
Comment (vigoux) [inline keywords.js:461]: I don't think those are `edit` variants, but we'll accept them anyway.

Comment (Fymyte) [reply keywords.js:461]: from doc:
```
:ex [++opt] [+cmd] [file]
			Same as |:edit|.
```
So at least ex is the same.
For visual, I don't think it's possible to use it in Ex-mode in scripts, and if not in Ex-mode, it is the same as `:edit:`
Same goes for `:view`

Comment (vigoux) [reply keywords.js:461]: 'kay may bad then

File: grammar.js [hunk 2]
```diff
@@ -167,68 +238,14 @@ module.exports = grammar({
...
-        "utf-32le",
-        "default"
-      ),
+    file_format: ($) => choice_arr(FILE_FORMAT),
+    _immediate_file_format: ($) =>
+      alias(choice_arr(FILE_FORMAT, (is_immediate = true)), $.file_format),
+
+    encoding: ($) => choice(...ENCODING),
+    _immediate_encoding: ($) =>
+      alias(choice(...ENCODING.map(token.immediate)), $.encoding),
```
Comment (Fymyte) [inline grammar.js:247]: I would like to know if this is ok for you, and if you prefert to use a function for that or juste use as for encoding

**Pull request merged**